files/default/neutron-ha-tool.py

```diff
@@ -22,6 +22,7 @@
 import argparse
 import random
 import time
+import retrying
 from logging.handlers import SysLogHandler
 from collections import OrderedDict
```

Comment:
Notice that for cloud5 this import line should be:
`from ordereddict import OrderedDict`